benchmark/cluster/echo.js

```diff
@@ -7,16 +7,25 @@ if (cluster.isMaster) {
     workers: [1],
     payload: ['string', 'object'],
     sendsPerBroadcast: [1, 10],
+    advancedSerialization: ['yes', 'no'],
```

Comment:
why not just `true, false`?

Reply:
The benchmark code doesn’t like anything except numbers and strings – this could obviously also be the strings `'true'` and `'false'` if that’s what you’re asking for. I don’t have a strong opinion on that, I went with what felt right to me.

Reply:
no I just didn't realize the benchmark didn't support booleans.